staff/francesc-destruels/youtube-channel-template/index.html

```diff
@@ -0,0 +1,134 @@
+        </div>
+
+        <div class="userHeader">
+            <img class="imgN1" alt="hectorito"
+                src="https://yt3.ggpht.com/a-/AAuE7mBFySqjmFC2dHbOCxTmA66tyAww1XJw55Cyzw=s288-mo-c-c0xffffffff-rj-k-no">
+            <a class="textN"><strong>Hache</strong></a>
+        </div>
+
+        <nav class="linkNavGrid">
+            <div class="lnGrid">
+
+                <div class="ln1">
+                    <a class="textN"></a>HOME</a>
+                </div>
+
+                <div class="ln2">
+                    <a class="textN"></a>VIDEOS</a>
+                </div>
+
+                <div class="ln3">
+                    <a class="textN"></a>PLAY LIST</a>
+                </div>
+                <div class="ln4">
+                    <i class="fas fa-angle-down"></i>
+                </div>
+
+            </div>
+        </nav>
+
+        <div class="videos">
+            <section class="videoGrid">
+                <img class="v1" alt="deMierda" src="https://i.ytimg.com/vi_webp/ySyR59Or754/sddefault.webp?v=59fa2a65">
+                <p class="t1">
+                    Hola<br /> Este video esta muy<br /> chulo!
+                </p>
+                <i class="fas fa-ellipsis-v i1"></i>
+
+                <img class="v2" alt="deMierda" src="https://i.ytimg.com/vi_webp/ySyR59Or754/sddefault.webp?v=59fa2a65">
+                <p class="t2">
+                    Hola<br /> Este video esta muy<br /> chulo!
+                </p>
+                <i class="fas fa-ellipsis-v i2"></i>
+
+                <img class="v3" alt="deMierda" src="https://i.ytimg.com/vi_webp/ySyR59Or754/sddefault.webp?v=59fa2a65">
+                <p class="t3">
+                    Hola<br /> Este video esta muy<br /> chulo!
+                </p>
+                <i class="fas fa-ellipsis-v i3"></i>
+
+                <img class="v4" alt="deMierda" src="https://i.ytimg.com/vi_webp/ySyR59Or754/sddefault.webp?v=59fa2a65">
+                <p class="t4">
+                    Hola<br /> Este video esta muy<br /> chulo!
+                </p>
+                <i class="fas fa-ellipsis-v i4"></i>
+
+                <img class="v5" alt="deMierda" src="https://i.ytimg.com/vi_webp/ySyR59Or754/sddefault.webp?v=59fa2a65">
+                <p class="t5">
+                    Hola<br /> Este video esta muy<br /> chulo!
+                </p>
+                <i class="fas fa-ellipsis-v i5"></i>
+
+                <img class="v6" alt="deMierda" src="https://i.ytimg.com/vi_webp/ySyR59Or754/sddefault.webp?v=59fa2a65">
+                <p class="t6">
+                    Hola<br /> Este video esta muy<br /> chulo!
+                </p>
+                <i class="fas fa-ellipsis-v i6"></i>
+            </section>
+
+        </div>
+        <div class="links linksGrind">
+
+                <img id="img" class="linksPics" alt="icon"
```

Comment:
@FranDRoss you can't use the same id in different components

Reply:
I am not using it at all, I forgot to erase it